From 7c347c4e46bb7d2899fe093ff5795c43916ab458 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Wed, 2 May 2012 14:22:27 +1000
Subject: [PATCH] conf: add Intuos4 WL (PTK-540WL) to fdi file

4709394d76267df05f8dfa7f13e6beb1f9c7394f added it to the xorg.conf.d
snippet. For those stuck on HAL add it to the fdi file too.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 conf/wacom.fdi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/wacom.fdi b/conf/wacom.fdi
index f35d6a5..70304d2 100644
--- a/conf/wacom.fdi
+++ b/conf/wacom.fdi
@@ -3,7 +3,7 @@
 <deviceinfo version="0.2">
   <device>
     <match key="info.category" contains="input">
-      <match key="info.product" contains_outof="Wacom;WALTOP;WACOM">
+      <match key="info.product" contains_outof="Wacom;WALTOP;WACOM;PTK-540WL">
 	<merge key="input.x11_driver" type="string">wacom</merge>
       </match>
     </match>
-- 
1.7.10

